packages/stark-ui/src/modules/table/components/column.component.ts

```diff
@@ -0,0 +1,170 @@
+import {
+	Component,
+	ContentChild,
+	EventEmitter,
+	HostBinding,
+	Input,
+	Output,
+	TemplateRef,
+	ViewChild,
+	ViewEncapsulation
+} from "@angular/core";
+import { MatColumnDef } from "@angular/material/table";
+
+/**
+ * @ignore
+ */
+const _get: Function = require("lodash/get");
+
+/**
+ * Column sorting directions of the StarkTableComponent
+ */
+export type StarkTableColumnSortingDirection = "asc" | "desc" | "";
+
+/**
+ * Component to display a column inside the StarkTableComponent
+ */
+@Component({
+	selector: "stark-table-column",
+	templateUrl: "./column.component.html",
+	encapsulation: ViewEncapsulation.None
+})
+export class StarkTableColumnComponent {
+	/**
+	 * Adds class="stark-table-column" attribute on the host component
+	 */
+	@HostBinding("class") public class: string = "stark-table-column";
+
+	/**
+	 * Name of the property that will be the source of the column.
+	 */
+	@Input()
+	public get name(): string {
```

Comment:
Indeed, it has the same functionality. It is done like this for the moment because this is just the initial implementation of the table, we will see if we need to refactor this when we implement the rest of functionalities..